### PR TITLE
add untangled pool

### PR DESCRIPTION
**Pull request by jpangelle**
Add untangled pool

To do:

- [x] add docusign template id's
- [x] add slack channel id

### CODE REVIEW

File: metadata/0x9De3064f49696a25066252C35ede68850EA33BF8.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "slug": "up-series-1",
+    "description": "UP Series LLC is a Digital Credit Fund set up by Untangled Finance, a digital securitisation platform regulated in the UK. The first Series is set up to invest in Trade Finance Assets (TFA). TFA are short term, self-liquidating invoice-based financings in both domestic and cross border trade. TFA traditionally have a very low default rate. This Series will invest in TFA from multiple asset originators. The originators have good underwriting records with zero/low default rates to date. All asset originators are subject to due diligence by Untangled.",
+    "media": {
+      "logo": "https://storage.googleapis.com/tinlake/pool-media/untangled-pool-1/logo.png",
+      "icon": "https://storage.googleapis.com/tinlake/pool-media/untangled-pool-1/icon.svg",
+      "drop": "https://storage.googleapis.com/tinlake/pool-media/untangled-pool-1/drop.svg",
+      "tin": "https://storage.googleapis.com/tinlake/pool-media/untangled-pool-1/tin.svg"
+    },
+    "asset": "Trade Finance",
+    "assetMaturity": "180 to 270 days",
```
Comment (offerijns) [inline metadata/0x9De3064f49696a25066252C35ede68850EA33BF8.json:39]:
```suggestion
    "assetMaturity": "6 to 9 months",
```

File: metadata/0x9De3064f49696a25066252C35ede68850EA33BF8.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "attributes": {
+      "Issuer": "UP Series LLC",
+      "Links": {
+        "Executive Summary": "https://storage.googleapis.com/tinlake/docs/summaries/UP1.pdf",
+        "Website": "https://untangled.finance/"
```
Comment (offerijns) [inline metadata/0x9De3064f49696a25066252C35ede68850EA33BF8.json:44]: Looks like the Forum Discussion is already up, either https://gov.centrifuge.io/t/issuer-untangled-finance-diversified-trade-finance-fund/2769/2 or https://gov.centrifuge.io/t/untangled-digital-credit-fund/2768

File: profiles/0x9De3064f49696a25066252C35ede68850EA33BF8.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "issuer": {
+    "name": "UP Series LLC",
+    "email": "contact@untangled.finance",
+    "restrictedCountryCodes": [],
+    "minInvestmentCurrency": "5000000000000000000000",
+    "nonSolicitationNotice": "non-us"
+  },
+  "bot": {
+    "channelId": "xx"
```
Comment (offerijns) [inline profiles/0x9De3064f49696a25066252C35ede68850EA33BF8.json:26]:
```suggestion
    "channelId": "C0256C8N924"
```

**Review by offerijns — CHANGES_REQUESTED** [review]

File: metadata/0x9De3064f49696a25066252C35ede68850EA33BF8.json (hunk 2 above)
Comment (jpangelle) [reply metadata/0x9De3064f49696a25066252C35ede68850EA33BF8.json:45]: added 👍

File: metadata/0x9De3064f49696a25066252C35ede68850EA33BF8.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    "JUNIOR_OPERATOR": "0x2840292eeC553c5816e7715a1427BfeD69EBF82E",
+    "SENIOR_OPERATOR": "0x2B1448781b53fd399f380a31FBa9f839e04889fA",
+    "JUNIOR_TRANCHE": "0x3Bb758A44034BDDFe44D50bA0eDE962B699665Fd",
+    "SENIOR_TRANCHE": "0xe66162910e6d28965D1A90bB15Bb0E861745115F",
+    "JUNIOR_TOKEN": "0x0097f415bf4C1B5d1145896D7de74f3f767d5A31",
+    "SENIOR_TOKEN": "0x42A95D51a7bd4Ef4c3558E638F09268f4DE726c2",
+    "JUNIOR_MEMBERLIST": "0xF32bAC8a57f2e7354bb82D05DCEe658A5Db5981a",
+    "SENIOR_MEMBERLIST": "0xe9Eaf8054476F73782B408b4F20CDfA277390817",
+    "ASSESSOR": "0xdBBa712f10061A3c7619Dc46852d8399F33C9fC7",
+    "ASSESSOR_ADMIN": "0xc9caE66106B64D841ac5CB862d482704569DD52d",
```
Comment (offerijns) [inline metadata/0x9De3064f49696a25066252C35ede68850EA33BF8.json:21]:
```suggestion
    "POOL_ADMIN": "0x545D98baa87B9C13305b081ef102ECb6F4F6f3C0",
```

**Review by offerijns — CHANGES_REQUESTED** [review]

**Review by offerijns — APPROVED** [review]

**Pull request merged**